tests/test-unix.js

```diff
@@ -27,7 +31,12 @@ tape('setup', function(t) {
 })
 
 tape('unix socket connection', function(t) {
```

Comment:
I would like to have two separate tests, the original one without querystring and a new one with querystring. That way if someone removes the unix bits from the init function the tests will throw an error.